lib/datasource/index.ts

```diff
@@ -124,3 +126,51 @@ export function getDigest(
     value
   );
 }
+export function sanitizeSourceUrl(rawSourceUrl) {
+  let repoUrlTmp: string;
+  if (!rawSourceUrl.endsWith('.git')) {
+    logger.debug(`${rawSourceUrl} is not a valid git url.`);
+    return null;
+  }
+  const git_info = hostedGitInfo.fromUrl(rawSourceUrl, { noGitPlus: true });
+  if (git_info === undefined) {
```

Comment:
switch this logic around, e.g. `if (gitInfo) {`

Reply:
Fixed.